cirq/ops/gate_operation.py

```diff
@@ -121,7 +121,12 @@ def _has_mixture_(self) -> bool:
         return protocols.has_mixture(self.gate)
 
     def _mixture_(self) -> Sequence[Tuple[float, Any]]:
-        return protocols.mixture(self.gate, NotImplemented)
+        mixture = protocols.mixture(self.gate, NotImplemented)
+        return ([(
+            p,
+            g(*self.qubits),
+        ) for p, g in mixture]
+                if mixture is not NotImplemented else NotImplemented)
```

Comment:
Split the `if NotImplemented return NotImplemented` condition out into a statement, so that yapf doesn't do such a terrible job here.